custom_components/coway/sensor.py

```diff
@@ -34,12 +36,14 @@ async def async_setup_entry(
     sensors = []
 
     for purifier_id, purifier_data in coordinator.data.purifiers.items():
+            _LOGGER.debug(f'PURIFIER DATA for {purifier_id}: {purifier_data}\n')
```

Comment:
See note above regarding LOGGER to be used. Also, no line break is needed as we are iterating through the purifiers here so each purifier will have its own debug line already.